models/user/user.go

```diff
@@ -413,6 +415,34 @@ func (u *User) DisplayName() string {
 	return u.Name
 }
 
+var emailToReplacer = strings.NewReplacer(
```

Comment:
`emailNameReplacer`?
I don't see any reason why we should only use it for `To`.

Reply:
Right now we do, if it gets reused some can just rename it